### PR TITLE
Kostik/refractor query system workflow/aact 570

### PR DESCRIPTION
**Pull request by kostik700015**
That PR adds changes listed in the description of ticket AACT-570.

Description:

1. In connect change the description for “Browser” to “Click here to run queries on the AACT data from your web browser”

2. Change the title to “User Submitted Queries”

3. Below the title we want to add the disclaimer: “These queries were submitted by aact users and have not been verified by the ctti aact team.”

4. Add a link from the saved_queries page to the instructions page

5. In the instructions page change the screenshots to show the real list of queries.

6. In the background jobs index make the title “Query Results”

7. Add a link to the background_jobs from the saved queries index page

### CODE REVIEW

File: app/views/background_jobs/index.html.erb
Comment (Javier-Jimenez-18) [inline app/views/background_jobs/index.html.erb:1]: Looks fine. You should provide a screenshot of the index page to show new title.

File: app/views/pages/connect.html.erb [hunk 1]
```diff
@@ -12,7 +12,7 @@
   <section class="connect">
   <div class="connectionTypes">
     <a href="/saved_query_doc" class="connectionCard">
-      <h4>Browser</h4>
+      <h4>Click here to run queries on the AACT data from your web browser</h4>
```
Comment (Javier-Jimenez-18) [inline app/views/pages/connect.html.erb:15]: This should be "Browser".

File: app/views/pages/connect.html.erb [hunk 2]
```diff
@@ -12,7 +12,7 @@
...
       <p>
         Click here for instructions about how to use your browser to analyze the AACT data directly.
```
Comment (Javier-Jimenez-18) [inline app/views/pages/connect.html.erb:17]: This should be "Click here to run queries on the AACT data from your web browser."

File: app/views/pages/connect.html.erb
Comment (Javier-Jimenez-18) [inline app/views/pages/connect.html.erb:1]: Also provide a screenshot of the connect view page.

File: app/views/saved_queries/index.html.erb
Comment (Javier-Jimenez-18) [inline app/views/saved_queries/index.html.erb:1]: Looks fine. Again, just provide a screenshot of the index page.

**Review by Javier-Jimenez-18 — CHANGES_REQUESTED** [review]
Overall good job. Just a couple of changes.

File: app/views/pages/connect.html.erb
Comment (kostik700015) [reply app/views/pages/connect.html.erb:1]: Thank you, will do

File: app/views/pages/connect.html.erb (hunk 1 above)
Comment (kostik700015) [reply app/views/pages/connect.html.erb:15]: It says "In connect change the description for “Browser” to “Click here to run queries on the AACT data from your web browser”"

Comment (kostik700015) [reply app/views/pages/connect.html.erb]: Fixed, thank you!

**Review by micronix — APPROVED** [review]
Looks good

**Pull request merged**